### PR TITLE
Add informations in areas page

### PR DESCRIPTION
**Pull request by PatrykSzwer**
For #133 

In the future, this one additional line can be used as a tooltip for Attach URI column name in tables, when https://github.com/Juicy/juicy-tooltip is done.

### CODE REVIEW

File: src/BlendingEditor/wwwroot/BlendingEditor/views/AreasPage.html [hunk 1]
```diff
@@ -14,96 +15,96 @@ <h2 slot="blendingeditor/areas-layout-header">Layout - {{model.Layout.Name}}</h2
...
-                                        Items ({{area.AreaItems.length}})
-                                    </template>
-                                    <template is="dom-if" if="{{!area.AreaItems.length}}">
-                                        No items
+                            <td colspan="8" style="background-color: var(--mouse);">
+                                <table class="blendingeditor-table" style="width: initial">
+                                    <thead>
+                                    <tr>
+                                        <th>Order</th>
+                                        <th>Blend URI</th>
```
Comment (warpech) [inline src/BlendingEditor/wwwroot/BlendingEditor/views/AreasPage.html]: Pls change the label to "Attach URI"

File: src/BlendingEditor/wwwroot/BlendingEditor/views/AreasPage.html [hunk 2]
```diff
@@ -5,6 +5,7 @@
     <dom-bind>
         <template is="dom-bind">
             <h2 slot="blendingeditor/areas-layout-header">Layout - {{model.Layout.Name}}</h2>
+            <div slot="blendingeditor/areas-summary">An area item defines an additional URI that should be requested and inserted into a specific layout area. Area item attach URI parameters <b>must</b> be wrapped in curly braces e.g. <code>/people/person/{123}</code></div>
```
Comment (warpech) [inline src/BlendingEditor/wwwroot/BlendingEditor/views/AreasPage.html]: Please move the second sentence as tooltip (`title=""`) on the "Attach URI" column.

Please change all `<th>` elements that have `title=""` to have `cursor: help` and an emoji ℹ️

File: src/BlendingEditor/wwwroot/BlendingEditor/views/AreasPage.html (hunk 1 above)
Comment (PatrykSzwer) [reply src/BlendingEditor/wwwroot/BlendingEditor/views/AreasPage.html]: For some reason this change wasn't included in commit. It's already done.

**Comment by PatrykSzwer** [comment]
I've done requested changes. Please take a look.

**Pull request merged**